src/main/java/randoop/condition/OperationConditions.java

```diff
@@ -0,0 +1,173 @@
+package randoop.condition;
+
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Represents all conditions on an operation including pre-, post-, and throws-conditions defined on
+ * this operation and inherited from supertypes. Does not represent the operation itself.
+ *
+ * <p>For an operation that is a method, the {@link OperationConditions} form an arbitrary directed
```

Comment:
This paragraph is inconsistent with the previous paragraph.  The previous paragraph indicates that the abstraction represented is the conditions, so how they are represented is an implementation detail.  This suggests differently.  Please make them consistent, so the documentation says how clients should view this code.